### PR TITLE
Fix rel nightly test by extending catchpoint round timeout

### PR DESCRIPTION
**Pull request by egieseke**
## Summary

Fix catchpoint unit test by extending timeout from 10 to 60 seconds.

## Test Plan

Tested locally.

### CODE REVIEW

**Review by algobarb — APPROVED** [review]
This looks good based on our conversation.

File: test/e2e-go/features/catchup/catchpointCatchup_test.go [hunk 1]
```diff
@@ -200,7 +200,7 @@ func TestBasicCatchpointCatchup(t *testing.T) {
 	targetRound = uint64(1)
 	log.Infof("Second node catching up to round 1")
 	for {
-		err = fixture.ClientWaitForRound(secondNodeRestClient, currentRound, 10000*time.Millisecond)
+		err = fixture.ClientWaitForRound(secondNodeRestClient, currentRound, 60*time.Second)
```
Comment (tsachiherman) [inline test/e2e-go/features/catchup/catchpointCatchup_test.go:203]: Assuming that this issue is reproducible, we should figure out why it's happening. Just masking this issue would make the platform less reliable ( i.e. this type of waits would, for example, mask off a 21 second rounds blocks ). Maybe try to conduct a binary search to find where the regression point was ?

Comment (onetechnical) [reply test/e2e-go/features/catchup/catchpointCatchup_test.go:203]: +1 on this. I'm a little worried this is highlighting a bigger issue, especially if it is a consistent failure.

**Comment by egieseke** [comment]
First commit that caused this

**Pull request merged**